### PR TITLE
feat: signup for newsletter on blog

### PR DESCRIPTION
**Pull request by cabreraalex**

### CODE REVIEW

**Comment by Sparkier** [comment]
CleanShot 2024-01-09 at 20 07 17@2x

can we make this full width?

File: docusaurus.config.js [hunk 1]
```diff
@@ -41,7 +41,9 @@ const config = {
           sidebarPath: require.resolve("./sidebars.js"),
         },
         blog: {
-          showReadingTime: true,
```
Comment (Sparkier) [inline docusaurus.config.js:44]: Don't we need this for the duration display??

**Review by Sparkier — CHANGES_REQUESTED** [review]
see comment

File: docusaurus.config.js (hunk 1 above)
Comment (cabreraalex) [reply docusaurus.config.js:44]: It's default true

**Comment by cabreraalex** [comment]
I couldn't figure out how to make it full width sadly :/

**Comment by cabreraalex** [comment]
Okay check now, it's slightly better with same BG

**Review by Sparkier — APPROVED** [review]

**Pull request merged**